래비/1주차, 2주차/style.css

```
@@ -0,0 +1,182 @@
+body {
+    background-image: url('img.svg');
+    background-size: 100%;
+}
+
+/* main Page */
+
+#App {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+}
+
+#loginForm {
+    background-color: #93959F;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 30px 200px;
+    border-radius: 15px;
+}
+
+#loginForm > h1 {
+    padding-left: 100px;
+    padding-right: 100px;
+    padding-bottom: 10px;
```

padding을 네 면중 세 면에만 주고 싶다면 이렇게 한 줄로 줄일 수 있습니다~
참고: https://developer.mozilla.org/ko/docs/Web/CSS/padding

```suggestion
    padding: 100px 10px 100px;
```

---

감사합니다! ㅎㅎ